activerecord/lib/active_record/railtie.rb

```diff
@@ -246,6 +254,7 @@ class Railtie < Rails::Railtie # :nodoc:
           :query_log_tags,
           :cache_query_log_tags,
           :sqlite3_production_warning,
+          :sqlite3_strict_mode,
```

Comment:
`sqlite3_strict_mode` means to me (at list having no knowledge about internals) there is some STRICT MODE in Sqlite3 itslef. But as mentioned in other comments, it is actually Sqlite3Adapter strict mode. Maybe it would fit better to call it `sqlite3_adapter_strict_mode`. :thinking: